src/app/ApplicationTemplate.Droid/Properties/AndroidManifest.xml

```diff
@@ -13,4 +13,4 @@
 			<meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/files" />
```

Comment:
Will this be remove in another PR?

Reply:
Yes the whole Android Xamarin project, it's still there so we can compare in case something is missing or not working.